public/app/plugins/panel/graph3/GraphPanel.tsx

```diff
@@ -233,6 +234,7 @@ export const GraphPanel: React.FC<GraphPanelProps> = ({
             <ZoomPlugin onZoom={onChangeTimeRange} />
             <ContextMenuPlugin />
 
+            {data.annotations && <ExemplarsPlugin exemplars={data.annotations} timeZone={timeZone} />}
```

Comment:
Love this plugin architecture, amazing work with this!! ❤️

Reply:
yea, really nice!